socorro/unittest/cron/jobs/test_bugzilla.py

```diff
@@ -66,9 +63,9 @@ def tearDown(self):
         super(IntegrationTestBugzilla, self).tearDown()
 
     def _setup_config_manager(self, days_into_past):
-        return get_config_manager_for_crontabber(
-            jobs='socorro.cron.jobs.bugzilla.BugzillaCronApp|1d',
-            overrides={
+        return super(IntegrationTestBugzilla, self)._setup_config_manager(
+            jobs_string='socorro.cron.jobs.bugzilla.BugzillaCronApp|1d',
+            extra_value_source={
                 'crontabber.class-BugzillaCronApp.days_into_past': days_into_past,
```

Comment:
This test was doing setup differently than all the other crontabber tests. I don't know why. Though this code looks familiar, so maybe it's something I broke/fixed recently.